node/src/BaseClient.ts

```diff
@@ -1882,13 +1904,11 @@ export class BaseClient {
      * of the list stored at `destination` depending on `whereTo`.
      * `BLMOVE` is the blocking variant of {@link lmove}.
      *
+     * @see {@link https://valkey.io/commands/blmove/|valkey.io} for details.
      * @remarks
      * 1. When in cluster mode, both `source` and `destination` must map to the same hash slot.
      * 2. `BLMOVE` is a client blocking command, see https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best practices.
-     *
-     * See https://valkey.io/commands/blmove/ for details.
-     *
-     * Since Valkey version 6.2.0.
+     * @remarks Since Valkey version 6.2.0.
```

Comment:
Probably 
```suggestion
     * 3. Since Valkey version 6.2.0.
```
Because we already have `@remarks` tag there

Reply:
Check other commands (blocking & multi-key) for such situation.

Reply:
Is there a chance to use `@since` tag - it looks more natural even if it is not built-in?

Reply:
I'm looking into adding custom tag, but tsdoc doesn't seem to have that option

Reply:
Adding a second @remarks tag renders fine

Reply:
I guess tsdoc doesn't care about custom tags, but eslint does.
Should we break then existing lists in remarks then for consistency?

Reply:
custom tags is not available yet: 
https://github.com/microsoft/tsdoc/issues/21

Reply:
> Should we break then existing lists in remarks then for consistency?

If they aren't related, then yet.  Will fix...

Reply:
> custom tags is not available yet: [microsoft/tsdoc#21](https://github.com/microsoft/tsdoc/issues/21)

https://tsdoc.org/pages/packages/tsdoc-config/
We use `@internal` a lot, tsdoc ignores them. `eslint` looks more sensitive for such things. He is a spotless's lil bro.

Reply:
`@internal` is a supported tag: https://tsdoc.org/pages/tags/internal/